eXo/Sources/Tool.swift

```diff
@@ -29,4 +29,27 @@ class Tool {
         }
     }
 
+    //Remove the protocol (http:// or https://) of a URL in string
+    static func stringURLWithoutProtocol (stringURLWithProtocol : String) -> String {
+        var stringURLWithoutProtocol = stringURLWithProtocol.stringByReplacingOccurrencesOfString("http://", withString: "");
+        stringURLWithoutProtocol = stringURLWithoutProtocol.stringByReplacingOccurrencesOfString("https://", withString: "")
+        return stringURLWithoutProtocol
+    }
+    
+    struct ShortcutTitleConfig {
+        static let maximunCharacters:Int = 30
+        static let prefixCharacters:Int = 9
+        static let subfixCharacters:Int = 15
+    }
+
+    static func titleForServer (server : Server) -> String {
```

Comment:
Is it possible / does it make sense to put this method in the `Server` object ?
Se we could call `server.title` ...